doc/install/install-ceph-gateway.rst

```diff
@@ -173,6 +173,11 @@ non-ssl connections are hosted by a single rgw instance. For eg::
  [client.rgw.gateway-node1]
  rgw_frontends = civetweb port=80+443s ssl_certificate=/etc/ceph/private/keyandcert.pem
 
+Additional Civetweb Configuration Options 
+----------------------------------------- 
+Some additional configuration options can be adjusted for the embedded Civetweb web server 
+in the **Client RADOS Gateway** section of the ``/etc/ceph/ceph.conf`` file. 
+A list of supported options, including an example, can be found in the `RadosGW frontends site`_. 
```

Comment:
the docs are pretty consistent about using the name `Ceph Object Gateway` for radosgw, and this paragraph uses two other names

it seems a bit long-winded to specify which `section of the /etc/ceph/ceph.conf file.` here, when there's another ceph.conf example just above this paragraph - i think `in ceph.conf` should be plenty

the title of radosgw/frontends.rst is `HTTP Frontends`, so it would make sense for this to say `can be found in HTTP Frontends.`

Reply:
> 
> 
> the docs are pretty consistent about using the name `Ceph Object Gateway` for radosgw, and this paragraph uses two other names
> 
> it seems a bit long-winded to specify which `section of the /etc/ceph/ceph.conf file.` here, when there's another ceph.conf example just above this paragraph - i think `in ceph.conf` should be plenty
> 
> the title of radosgw/frontends.rst is `HTTP Frontends`, so it would make sense for this to say `can be found in HTTP Frontends.`

@cbodley  ok, hope that's what you asked for...